### ISSUE TITLE
Add button to show extra content when the option to show it unticked

### ISSUE THREAD

**Issue by Xerkus** [android-password-store/Android-Password-Store#502]
Currently, when "show extra content" is selected, extra content is displayed on the screen in clear text when you unlock the password.
Problem with that is that extra content can be used for sensitive stuff like 2fa recovery codes (I use hardware token so I consider it secure enough for most less important accounts, but that is beside the point)
Unselecting that option leaves no button to view it or even mention it is there. The only way to access it then is to edit entry. That can lead to the content erroneously changed when you only want to view/copy it. 

TLDR: I would like to see "Show extra content" button under the "show password" button when extra content is available and setting to show it is unselected.

**Comment by emcorrales** [comment]
Hello, I created a PR for this. https://github.com/zeapo/Android-Password-Store/pull/510

**Comment by emcorrales** [comment]
I opened a PR for this.

**Comment by msfjarvis** [comment]
#510 fixed this but did not mark the issue for closing.
